zerver/tests/test_narrow.py

```diff
@@ -368,17 +426,21 @@ def test_successful_get_old_messages(self):
         narrow = [['pm-with', 'othello@zulip.com']]
         self.get_and_check_messages(dict(narrow=ujson.dumps(narrow)))
 
-        narrow = [dict(operator='pm-with', operand='othello@zulip.com')]
+        narrow = [dict(operator='pm-with', operand='othello@zulip.com')] # type: ignore
```

Comment:
Hmm. If you can come up with a descriptive name for the variable then change the variable name otherwise pass the data directly.